rxandroidble/src/main/java/com/polidea/rxandroidble2/scan/ScanRecord.java

```diff
@@ -22,6 +24,14 @@
     @Nullable
     List<ParcelUuid> getServiceUuids();
 
+    /**
+     * Returns a list of service solicitation UUIDs within the advertisement that are used to identify the
+     * bluetooth GATT services the peripheral requires on the Central.
+     */
+    @RequiresApi(29 /* Build.VERSION_CODES.Q */)
```

Comment:
Correct me if I am wrong — but it is possible to backport this for all APIs as this data is advertised in BT 4.0 compatible way?

Reply:
I see you fixed this